packages/policy/test/test.cli.ts

```diff
@@ -112,4 +113,25 @@ describe('cli', () => {
     await cli.main(m);
     assert.ok(exportStub.calledOnce);
   });
+
+  it('should attempt to file an issue if asked nicely', async () => {
+    process.env.GH_TOKEN = 'token';
```

Comment:
Will this cause weirdness with future tests?

Reply:
It doesn't, but just to be safe I came back around and cached the original value, and reset it after the tests run.